text/0057-general-multiplication.md

```diff
@@ -0,0 +1,91 @@
+# General multiplication in Cedar
+
+## Related issues and PRs
+
+- Reference Issues: related to [RFC 50]
```

Comment:
Is there a correctness issue that this relates to, e.g, relating to overflow and ordering issues that you hint at below? If so, reference that here.

Reply:
There is not currently a referenceable issue, beyond conversation on RFC 50